packages/nextjs/components/scaffold-eth/EtherInput.tsx

```diff
@@ -2,7 +2,35 @@ import React, { ChangeEvent, useEffect, useState } from "react";
 import { useAppStore } from "~~/services/store/store";
 import { ArrowsRightLeftIcon } from "@heroicons/react/24/outline";
 
-type TEtherInputProps = {
+function etherValueToDisplayValue(usdMode: boolean, etherValue: string, ethPrice: number) {
```

Comment:
since functions almost the same I'd propose something like this

```
function multiplyValue(applyMultiplying: boolean, value: string, multiplier: number) {
  if (applyMultiplying && multiplier) {
    const parsedEthValue = parseFloat(value);
    if (!Number.isNaN(parsedEthValue)) {
      return (parsedEthValue * multiplier).toString();
    }
  }

  return value;
}

function etherValueToDisplayValue(usdMode: boolean, etherValue: string, ethPrice: number) {
  return multiplyValue(usdMode, etherValue, ethPrice);
}

function displayValueToEtherValue(usdMode: boolean, displayValue: string, ethPrice: number) {
  return multiplyValue(usdMode, displayValue, 1 / ethPrice);
}
```

Reply:
Added a `toFixed(2)` for the usd display value, so not sure if we can still apply this suggestion in some way. Let me know if you have an idea.